### DISCUSSION TITLE
Will SigZoz works in a closed vpn without outside internet connectivity?

### DISCUSSION THREAD

**Discussion by praveeshvp** [SigNoz/signoz#2977] [Q&A]
This is Onboard ship use case where we wanted to deploy SigZoz for onboard APM. 
Onboard ship will not have internet connectivity but will have local connectivity with closed vpn. So wanted to SigZoz will work without internet connectivity on a on premise deployment. We will have connectivity with all applications which wanted to monitor on board but limited/no internet connectivity to external. Dynatrace is the APM tool using for some of ships but due to cost factor we are looking alternative.

Appreciate if anyone can answer to this question and let me know if you need more information on this.

**Comment by pranay01** [comment]
Hey @praveeshvp thanks for sharing question here.

Can you share more details on what you mean by onboard ship? Is it a marine ship or a space ship?

Ideally, if you run SigNoz within your VPN, you should be able to run it without need to be connected to the Internet.

**Reply by praveeshvp** [reply]
@pranay01 thanks a lot for the reply. 
This is cruise line, so connectivity will be via satellite which is ON & OFF scenario. Though limited internet connectivity available for Guest to use, the majority of the systems onboard will not have access to the internet. This is on premise deployment within VPN.
